docs/source/install/ewc_ha.rst

```diff
@@ -0,0 +1,286 @@
+
+st2auth
+_______
+All authentication is managed by ``st2auth`` service.
+K8s configuration includes a Pod Deployment backed by ``2`` replicas by default and Service of type ClusterIP listening on port ``9100``.
+Multiple st2auth processes can be behind a load balancer in an active-active configuration and you can increase number of replicas per your discretion.
+
+st2api
+______
+Service hosts the REST API endpoints that serve requests from WebUI, CLI, ChatOps and other st2 components.
+K8s configuration consists of Pod Deployment with ``2`` default replicas for HA and ClusterIP Service accepting HTTP requests on port ``9101``.
+Being one of the most important StackStorm services with a lot of logic involved,
+it's recommended to increase number of replicas to distribute the load if you'd plan increased processing environment.
+
+st2stream
+_________
+StackStorm st2stream - exposes a server-sent event stream, used by the clients like WebUI and ChatOps to receive updates from the st2stream server.
+Similar to st2auth and st2api, st2stream K8s configuration includes Pod Deployment with ``2`` replicas for HA (can be increased in ``values.yaml``)
+and ClusterIP Service listening on port ``9102``.
+
+st2rulesengine
+______________
+st2rulesengine evaluates rules when it sees new triggers and decides if new action execution should be requested.
+K8s config includes Pod Deployment with ``2`` (configurable) replicas by default for HA.
+
+st2timersengine
+_______________
+st2timersengine is responsible for scheduling all user specified `timers <https://docs.stackstorm.com/rules.html#timers>`_ aka st2 cron.
+Only single replica is created via K8s Deployment as timersengine can't work in active-active mode at the moment
+(multiple timers will produce duplicated events) and it relies on K8s failover/reschedule capabilities to address cases of process failure.
+
+st2workflowengine
+_________________
+st2workflowengine drives the execution of orquesta workflows and actually schedules actions to run by another component ``st2actionrunner``.
+Multiple st2workflowengine processes can run in active-active mode and so minimum ``2`` K8s Deployment replicas are created by default.
+All the workflow engine processes will share the load and pick up more work if one or more of the processes become available.
+
+.. note::
+  As Mistral is going to be deprecated and removed from StackStorm platform soon, Helm chart relies only on
+  :doc:`Orquesta st2workflowengine </orquesta/index>` as a new native workflow engine.
+
+st2notifier
+___________
+Multiple st2notifier processes can run in active-active mode, using connections to RabbitMQ and MongoDB and generating triggers based on
+action execution completion as well as doing action rescheduling.
+In an HA deployment minimum ``2`` replicas of st2notifier is running, requiring coordination backend, which is ``etcd`` in our case.
```

Comment:
In an HA deployment there must be a minimum of ``2`` replicas of st2notifier running, requiring a coordination backend, which in our case is ``etcd``.